src/applications/pokemon_summary_screen/window.c

```diff
@@ -26,177 +29,760 @@
-    [PSS_STATIC_WINDOW_17] = { 0x1, 0x10, 0x9, 0x6, 0x2, 0xF, 0x11D },
-    [PSS_STATIC_WINDOW_18] = { 0x1, 0x10, 0xB, 0x6, 0x2, 0xF, 0x129 },
-    [PSS_STATIC_WINDOW_19] = { 0x1, 0x10, 0xD, 0x6, 0x2, 0xF, 0x135 },
-    [PSS_STATIC_WINDOW_20] = { 0x1, 0x10, 0xF, 0x6, 0x2, 0xF, 0x141 },
-    [PSS_STATIC_WINDOW_21] = { 0x1, 0xE, 0x12, 0x6, 0x2, 0xF, 0x14D },
-    [PSS_STATIC_WINDOW_22] = { 0x1, 0xE, 0x14, 0x5, 0x2, 0xF, 0x159 },
-    [PSS_STATIC_WINDOW_23] = { 0x1, 0x1, 0x16, 0x6, 0x2, 0xF, 0x163 },
-    [PSS_STATIC_WINDOW_24] = { 0x1, 0x15, 0x14, 0x5, 0x2, 0xF, 0x16F },
-    [PSS_STATIC_WINDOW_25] = { 0x1, 0x1, 0x8, 0x6, 0x2, 0xF, 0x179 },
-    [PSS_WINDOW_MOVE_POWER] = { 0x1, 0x1, 0xA, 0x6, 0x2, 0xF, 0x185 },
-    [PSS_WINDOW_MOVE_ACCURACY] = { 0x1, 0x1, 0xC, 0x8, 0x2, 0xF, 0x191 },
-    [PSS_STATIC_WINDOW_28] = { 0x1, 0x12, 0xB, 0x9, 0x2, 0xF, 0x1A1 },
-    [PSS_STATIC_WINDOW_29] = { 0x1, 0x2, 0xD, 0xC, 0x2, 0xF, 0x1B3 },
-    [PSS_STATIC_WINDOW_30] = { 0x1, 0xE, 0x15, 0xC, 0x2, 0xF, 0x1CB },
-    [PSS_STATIC_WINDOW_31] = { 0x1, 0x1, 0x0, 0xB, 0x2, 0xF, 0x1E3 },
-    [PSS_WINDOW_ITEM_NAME] = { 0x1, 0x1, 0x16, 0xC, 0x2, 0xF, 0x1F9 },
-    [PSS_WINDOW_MON_LEVEL] = { 0x1, 0x1, 0x5, 0x6, 0x2, 0xF, 0x211 },
-    [PSS_WINDOW_MON_NICKNAME] = { 0x1, 0x3, 0x3, 0x9, 0x2, 0xF, 0x21D },
-    [PSS_WINDOW_BUTTON_PROMPT] = { 0x1, 0x1A, 0x0, 0x6, 0x2, 0xF, 0x22F }
+// todo: consolidate this with more generic ones if this scheme is used elsewhere
+enum PSSTextAlignment {
+    ALIGN_LEFT = 0,
+    ALIGN_RIGHT,
+    ALIGN_CENTER,
 };
 
-static const WindowTemplate Unk_020F4E20[] = {
-    { 0x1, 0x18, 0x5, 0x6, 0x2, 0xF, 0x23B },
-    { 0x1, 0x17, 0x7, 0x8, 0x2, 0xF, 0x247 },
-    { 0x1, 0x17, 0xB, 0x8, 0x2, 0xF, 0x257 },
-    { 0x1, 0x19, 0xD, 0x4, 0x2, 0xF, 0x267 },
-    { 0x1, 0x18, 0x11, 0x6, 0x2, 0xF, 0x271 },
-    { 0x1, 0x18, 0x15, 0x6, 0x2, 0xF, 0x27F }
+static void PrintStrbufToWindow(PokemonSummaryScreen *summaryScreen, Window *window, TextColor color, enum PSSTextAlignment alignment);
+static void PrintTextToStaticWindow(PokemonSummaryScreen *summaryScreen, enum PSSStaticWindow windowIndex, u32 entryID, TextColor color, enum PSSTextAlignment alignment);
+static void PrintStaticWindows(PokemonSummaryScreen *summaryScreen);
+static void PrintMoveNameAndPP(PokemonSummaryScreen *summaryScreen, u32 moveIndex);
+static void DrawInfoPageWindows(PokemonSummaryScreen *summaryScreen);
+static void DrawMemoPageWindows(PokemonSummaryScreen *summaryScreen);
+static void DrawSkillsPageWindows(PokemonSummaryScreen *summaryScreen);
+static void DrawConditionPageWindows(PokemonSummaryScreen *summaryScreen);
+static void DrawBattleMovesPageWindows(PokemonSummaryScreen *summaryScreen);
+static void DrawContestMovesPageWindows(PokemonSummaryScreen *summaryScreen);
+static void DrawRibbonsPageWindows(PokemonSummaryScreen *summaryScreen);
+static void DrawExitPageWindows(PokemonSummaryScreen *summaryScreen);
+
+// clang-format off
```

Comment:
**suggestion**: I presume that this is here to preserve the struct-initializer formatting; you can coerce `clang-format` into respecting that by having the final line of each struct-initializer end in a trailing comma, as well as having the final initializer of the array end with a trailing comma.

(I would argue that that is good style, anyways, regardless of if `clang-format` respects it or not)